src/validation/__tests__/KnownDirectives-test.js

```diff
@@ -5,12 +5,16 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+import { expect } from 'chai';
 import { describe, it } from 'mocha';
+import { parse } from '../../language';
 import { buildSchema } from '../../utilities';
+import { validate } from '../validate';
 import {
   expectPassesRule,
   expectFailsRule,
   expectSDLErrorsFromRule,
+  testSchema,
 } from './harness';
```

Comment:
@mjmahone You can reuse this change from your previous PR:
https://github.com/graphql/graphql-js/pull/1422/files#diff-43fa06a89eafc57a60a0108a96f1c502R424